camelot/view/controls/delegates/delegatemanager.py

```diff
@@ -52,8 +54,9 @@ def get_column_delegate(self, index):
             field_attributes = index.model().headerData(
                 column, Qt.Horizontal, FieldAttributesRole
             )
-            delegate = field_attributes['delegate'](parent=self, **field_attributes)
-            self.insert_column_delegate(column, delegate)
+            if field_attributes is not None:
```

Comment:
komen we in deze case ?

Reply:
Nee, met de fix van die test_view.py unit test is dit niet meer nodig.